src/RTT_Format_Reader/CellDefs.cc

```diff
@@ -71,38 +71,45 @@ void CellDefs::readEndKeyword(ifstream & meshfile)
     string dummyString;
 
     meshfile >> dummyString;
-    Insist(dummyString == "end_cell_defs", "Invalid mesh file: cell_defs block missing end");
-    std::getline(meshfile, dummyString);       // read and discard blank line.
+    Insist(dummyString == "end_cell_defs",
+           "Invalid mesh file: cell_defs block missing end");
+    std::getline(meshfile, dummyString); // read and discard blank line.
 }
 /*!
  * \brief Changes the cell definitions specified in the RTT_Format file
- *        to an alternative coordinate-system independent cell definition 
+ *        to an alternative coordinate-system independent cell definition
  *        (e.g., CYGNUS).
- * \param cell_side_types New side types for each of the existing cell 
+ * \param cell_side_types New side types for each of the existing cell
  *        definitions.
- * \param cell_ordered_sides New ordered sides for each of the existing cell 
+ * \param cell_ordered_sides New ordered sides for each of the existing cell
  *        definitions.
  */
 void CellDefs::redefineCellDefs(
     vector_vector_int const & cell_side_types,
-    std::vector< std::vector< std::vector< size_t > > >
-    const & cell_ordered_sides)
+    std::vector<std::vector<std::vector<size_t>>> const & cell_ordered_sides)
 {
-    Insist(cell_side_types.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition side types data.");
-    Insist(cell_ordered_sides.size()
-           == static_cast<size_t>(dims.get_ncell_defs()),
-	   "Error in supplied cell redefinition ordered side data.");
+    Insist(cell_side_types.size() == static_cast<size_t>(dims.get_ncell_defs()),
+           "Error in supplied cell redefinition side types data.");
+    Insist(cell_ordered_sides.size() ==
+               static_cast<size_t>(dims.get_ncell_defs()),
+           "Error in supplied cell redefinition ordered side data.");
+
+    for (size_t cd = 0; cd < static_cast<size_t>(dims.get_ncell_defs()); cd++)
```

Comment:
These same checks appear in the next block of code below. Did you mean to remove this for-loop?